templates/deploy-master-statefulset.yaml

```diff
@@ -168,6 +168,8 @@ spec:
           securityContext: {{- .Values.master.volumePermissions.containerSecurityContext | toYaml | nindent 12 }}
           {{- if .Values.master.volumePermissions.resources }}
           resources: {{- toYaml .Values.master.volumePermissions.resources | nindent 12 }}
+          {{- else if ne .Values.master.volumePermissions.resourcesPreset "none" }}
+          resources: {{- include "deploy.resources.preset" (dict "type" .Values.master.volumePermissions.resourcesPreset) | nindent 12 }}           
```

Comment:
Not sure about that we should change the resources on the init-containers, in case of large and other options it will need a lot of resources.
I think we need for init-containers to just use specific predefined values, and not use presets?

And what about other init-containers?